src/Hooks/Filters.php

```diff
@@ -432,7 +452,7 @@ public static function getSettingsLabels(string $type, string $key = 'title'): s
 			],
 			SettingsEnrichment::SETTINGS_TYPE_KEY => [
 				'title' => \__('Enrichment', 'eightshift-forms'),
-				'desc' => \__('Using saved URL parameters, track users even when they leave the site.', 'eightshift-forms'),
+				'desc' => \__('Using saved URL parameters and cookies to track users even when they leave the site.', 'eightshift-forms'),
```

Comment:
```suggestion
				'desc' => \__('Using saved URL parameters and cookies track users even when they leave the site.', 'eightshift-forms'),
```
It's "Use x _to_ y", but it's "Using x _(you can do)_ y" 😄